spec/models/stripe_wrapper_spec.rb

```diff
@@ -29,25 +28,81 @@
     context "when card is invalid" do
       let(:card_number) { '4000000000000002'}
 
-      it "does not charge the card" do
-        expect(charge.status).to eq(:error)
+      it "does not create the customer" do
+        expect(customer_creation).not_to be_successful
       end
 
       it "contains an error message" do
-        expect(charge.response.message).to be_present
-      end
-
-      describe "successful?" do
-        it "returns true if the charge is successful" do
-          expect(charge.successful?).to eq(false)
-        end
+        expect(customer_creation.response.message).to be_an_instance_of(String)
       end
 
       describe "error_message" do
         it "should return an error message when a charge is declined" do
-          expect(charge.error_message).to be_an_instance_of(String)
+          expect(customer_creation.error_message).to be_an_instance_of(String)
         end
       end
     end
   end
 end
+
+describe StripeWrapper::Card do
+  describe ".default_card(user)", :vcr do
+    let(:user) { Fabricate(:user, stripe_customer_id: generate_stripe_customer_id) }
+
+    it "retrieves the customer's default card", :vcr do
+      expect(StripeWrapper::Card.default(user).last4).to eq('4242')
+    end
+  end
+
+  describe ".update_current_card(options={})" do
+  end
+
+  describe ".create(options={})", :vcr do
+    let(:user) { Fabricate(:user, stripe_customer_id: generate_stripe_customer_id) }
+
+    context "when card is valid" do
+      let(:card_number) { '4012888888881881' }
+      let(:token) { generate_stripe_token }
+
+      it "successfully creates a new card" do
+        card_creation = StripeWrapper::Card.create(user: user, token: token)
+        expect(card_creation).to be_successful
+      end
+
+      it "deletes the customer's old card" do
+        card_creation = StripeWrapper::Card.create(user: user, token: token)
+        customer = StripeWrapper::Customer.retrieve(user)
+        expect(customer.sources.total_count).to eq(1)
```

Comment:
I'm not sure this expectation actually verifies anything was deleted. I think an inequality would be a better verification, if it is possible.